### PR TITLE
Fix failing tests after merging c3 changes

### PR DESCRIPTION
**Pull request by xBis7**
This PR is a draft because it's depending on https://github.com/xBis7/docker-setup-helper-scripts/pull/52

### CODE REVIEW

File: trino-hms-hdfs-ranger/big-data-c3-tests/trino-spark-tests/trino/test_8.sh [hunk 1]
```diff
@@ -49,4 +49,4 @@ expectedMsg="DROP TABLE"
 runTrino "$TRINO_USER1" "$command" "shouldPass" "$expectedMsg"
 
 # 'hdfs dfs -ls' and check data after drop.
-listContentsOnHdfsPath "$TRINO_HIVE_WAREHOUSE_DIR/gross_test.db" "shouldBeEmpty" "" "devpod"
+listContentsOnHdfsPath "$TRINO_HIVE_WAREHOUSE_DIR/gross_test.db" "shouldBeEmpty" "-" "devpod"
```
Comment (xBis7) [inline trino-hms-hdfs-ranger/big-data-c3-tests/trino-spark-tests/trino/test_8.sh:52]: Without this change, the `location` value, `devpod` goes into the `expectedOutput` variable.

Comment (GeorgeJahad) [reply trino-hms-hdfs-ranger/big-data-c3-tests/trino-spark-tests/trino/test_8.sh:52]: can we change "-" to something more readable like: "ignoreExpectedOutput"

Comment (xBis7) [reply trino-hms-hdfs-ranger/big-data-c3-tests/trino-spark-tests/trino/test_8.sh:52]: Yes, I'll make the change.

**Review by GeorgeJahad — APPROVED** [review]
looks good except for the single minor comment.  fix that, run the tests, confirm they still work and then merge please.

thanks!

**Comment by xBis7** [comment]
@GeorgeJahad Thanks for the review! 

I've run all tests both automated and load tests without issues.

**Pull request merged**